src/var/intervalSet.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
```

Comment:
OK, I can't change it because the regex for matching defines in `/var/*` modules expects no space: https://github.com/jquery/jquery/blob/8e111df641cca3e1b75b31a1971bfc89014b4ded/build/tasks/build.js#L14 (cc @timmywil)